ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -209,11 +209,12 @@ class ManagementActor(managers: ProcessingTypeDataProvider[DeploymentManager],
             ProcessStatus.simpleErrorMismatchDeployedVersion(ver.versionId, action.processVersionId, action.user, processState)
           case Some(ver) if ver.versionId == action.processVersionId =>
             state
+          case None if !state.isDeployed =>
```

Comment:
let's just change line 206 to `case _ if !state.isDeployed => `?

Reply:
yeah, i've been thinking about it ;)